recipes/azure-cosmos/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "azure-cosmos" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 081b3f592c3cf88c4d423fbb8184732cae0427cc5188cfed4835d9cdb64fa5f7
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
```

Comment:
`setuptools` must also be listed here as it is a runtime dependency to declare the namespace.